src/FormAdapter/StandardFormAdapter.php

```diff
@@ -79,6 +79,62 @@ public function toQuery($data, $formSettings)
         return $query;
     }
 
+    public function stringifyData($data, $page) : array
+    {
+        $dataString = [];
+        $formSettings = $page->settings()['form'];
+
+        if (!empty($data['q'])) {
+            $dataString['main_query'] = $data['q'];
+        }
+
+        if (!empty($data['filters']['queries'])) {
+            $filters = $this->stringifyFilters($data['filters']['queries'], $page, $data['filters']['match']);
+            if (!empty($filters)) {
+                $dataString["filters_subqueries"] = $filters;
+            }
+        }
+
+        foreach ($formSettings['elements'] ?? [] as $formElementData) {
+            $name = $formElementData['name'];
+            $apiManager = $this->getApiManager();
+            $formElement = $this->searchFormElementManager->get($name);
+            $dataString["form_element_$name"] = $formElement->stringifyData($data, $formElementData, $apiManager);
+        }
+
+        return $dataString;
+    }
+
+    public function stringifyFilters($queries, $page, $match = 'all')
+    {
+        $filters = [];
+        $formSettings = $page->settings()['form'];
+        $index = $page->index();
+
+        foreach ($queries as $query) {
+            if (!empty($query['queries'])) {
+                $filters[] = $this->stringifyFilters($query['queries'], $formSettings, $query['match']);
+            } elseif (!empty($query['term'])) {
+                $label = $this->getLabelForField($query['field'], $formSettings);
+                $operator = $page->index()->availableOperators($index)[$query['operator']];
+                $filters[] = sprintf("%s %s '%s'", $label, $operator['display_name'], $query['term']);
+            }
+        }
+
+        $separator = ($match === 'all') ? ' AND ' : ' OR '; // @translate
+        return count($filters) > 0 ? '(' . implode($separator, $filters) . ')' : '';
```

Comment:
I think `// @translate` only marks the last string on the line to be translatable.

But it doesn't really matter here anyway because `$separator` is included in another string, and that string won't be translatable